### ISSUE TITLE
test_win32events.py hangs on Windows with Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8626]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8626|
|Type|enhancement|
|Created|2016-07-19 04:15:45Z|
|Branch|https://github.com/twisted/twisted/tree/8626-rodrigc-win32reactor-py3|

I ran this command with Python 3.5 under Windows:

```
python bin\trial twisted.internet.test.test_win32events
```

and the test hung

<details><summary>Searchable metadata</summary>

```
trac-id__8626 8626
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8626_rodrigc_win32reactor_py3 8626-rodrigc-win32reactor-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468901745689370 1468901745689370
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/353

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 884af50645efd22eca9012b890c7e0b90f6e2689
```
#!CommitTicketReference repository="" revision="884af50645efd22eca9012b890c7e0b90f6e2689"
Merge pull request #353 from twisted/8626-rodrigc-win32reactor-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8626

Fix win32reactor on Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
